source/fe/mapping_q.cc

```diff
@@ -1085,6 +1085,26 @@ void MappingQ<dim,spacedim>::transform
   MappingQ1<dim,spacedim>::transform(input, output, *q1_data, mapping_type);
 }
 
+template<int dim, int spacedim>
+void MappingQ<dim,spacedim>::transform
+(const VectorSlice<const std::vector< DerivativeForm<2, dim, spacedim> > > input,
+ VectorSlice<std::vector<Tensor<3,spacedim> > >             output,
+ const typename Mapping<dim,spacedim>::InternalDataBase &internal,
+ const MappingType type) const
+{
+  this->MappingQ1<dim,spacedim>::transform(input, output, internal, type);
```

Comment:
@bangerth @tjhei Yes, the function body should be identical to the ones above it. The way I understand it, MappingQ delegates the computation to MappingQ1 since it is the same regardless of the mapping order. The difference comes in the mapping data, which would have been prepared beforehand to reflect the mapping order.